contacts24/models/record.py

```diff
@@ -39,48 +48,79 @@ class Record:
     def __init__(self, name: str):
         self.name = Name(name)
         self.phones: List[Phone] = []
+        self.emails: List[Email] = []
         self.birthday: Optional[Birthday] = None
+        self.address: Optional[Address] = None
 
     def add_birthday(self, birthday: str) -> None:
         self.birthday = Birthday(birthday)
 
+    def add_address(self, address: str) -> None:
+        self.address = Address(address)
+
     def add_phone(self, phone: str) -> None:
         new_phone = Phone(phone)
         if new_phone:
             self.phones.append(new_phone)
 
+    def add_email(self, email: str) -> None:
+        new_email = Email(email)
+        if new_email:
+            self.emails.append(new_email)
+
     def delete_phone(self, phone: str) -> None:
         self.phones = [p for p in self.phones if p.value != phone]
 
+    def delete_email(self, email: str) -> None:
+        self.emails = [p for p in self.emails if p.value != email]
+
     def edit_phone(self, old_phone: str, new_phone: str) -> None:
         for index, p in enumerate(self.phones):
             if p.value == old_phone:
                 self.phones[index] = Phone(new_phone)
                 break
 
+    def edit_email(self, old_email: str, new_email: str) -> None:
+        for index, p in enumerate(self.emails):
+            if p.value == old_email:
+                self.emails[index] = Email(new_email)
+                break
+
     def find_phone(self, phone: str) -> Optional[str]:
         for p in self.phones:
             if p.value == phone:
                 return p.value
         return None
 
+    def find_email(self, email: str) -> Optional[str]:
+        for p in self.emails:
+            if p.value == email:
+                return p.value
+        return None
+
     def to_dict(self):
```

Comment:
@IhorNi  as we have here serialization method to dict
should we keep one in `db.py` class OR I will switch `db.py` to use this method?

Reply:
@SavchukR We already pretty much screwed the idea of isolated delevepment of one module responsibility per a person)) But there is still a chance, I'd say that you switch to this `.to_dict()`, so that your part with `db.py` is finished and no further dependencies for you on that, just fo guys to extend `.to_dict()` as they go here and with Notes. What do you think?

Reply:
@IhorNi it's always question who should know about how to serialize object - object itself or class that need serialization.

Keep it as is for now
After PR is completed I will move serialization to the separate file `serialization.py` which will be responsible for this and consumed by `db.py`